rvps/src/reference_value.rs

```diff
@@ -75,13 +76,19 @@ fn default_version() -> String {
 }
 
 impl ReferenceValue {
-    pub fn new() -> Self {
-        ReferenceValue {
+    /// Create a new `ReferenceValue`, the `expired`
+    /// field's nanosecond will be set to 0. This avoid
+    /// a rare bug that when the nanosecond of the time
+    /// is not 0, the test case will fail.
+    pub fn new() -> Result<Self> {
+        Ok(ReferenceValue {
             version: REFERENCE_VALUE_VERSION.into(),
             name: String::new(),
-            expired: Utc::now(),
+            expired: Utc::now()
+                .with_nanosecond(0)
+                .ok_or_else(|| anyhow!("set nanosecond failed."))?,
```

Comment:
A generic comment on that commit: I do not think it's related to the PR it's included with. So please send a separate PR with only that commit, as a fix.

Reply:
Yea, thanks! I'm going to